src/CoreVideo/CVDisplayLink.cs

```diff
@@ -57,6 +58,64 @@ internal CVDisplayLink (IntPtr handle, bool owns)
 			this.handle = handle;
 		}
 
+		[Mac (12,0)]
+		[DllImport (Constants.CoreVideoLibrary)]
+		static extern CVReturn CVDisplayLinkCreateWithCGDisplay (uint displayId, out IntPtr displayLink);
+
+		[Mac (12,0)]
+		public static CVDisplayLink? FromDisplayId (uint displayId, out CVReturn error)
+		{
+			error = CVDisplayLinkCreateWithCGDisplay (displayId, out IntPtr handle);
+			if (error != 0)
+				return null;
+
+			return new CVDisplayLink (handle, true);
+		}
+
+		[Mac (12,0)]
+		public static CVDisplayLink? FromDisplayId (uint displayId)
+			=> FromDisplayId (displayId, out var _);
+
+		[Mac (12,0)]
+		[DllImport (Constants.CoreVideoLibrary)]
+		unsafe static extern CVReturn CVDisplayLinkCreateWithCGDisplays (uint* displayArray, nint count, out IntPtr displayLink);
```

Comment:
You can use arrays in the signature (and remove `unsafe`):

```suggestion
		static extern CVReturn CVDisplayLinkCreateWithCGDisplays (uint[] displayArray, nint count, out IntPtr displayLink);
```